px-modal.html

```diff
@@ -1,178 +1,351 @@
-`--px-modal-buttons`   | Style mixin to be applied to all of the buttons at the bottom of the overlay
-`--px-modal-positive-button`   | Style mixin to be applied specifically to the positive action button of the overlay
+`--px-modal-background-color` | Background color of the modal dialog
+`--px-modal-overlay-color`    | Color of the overlay that covers the screen behind the modal dialog
+`--px-modal-text-color`       | Text color for content set via. properties (slotted content will not use this color)
 
 @element px-modal
-@blurb Element providing means to open and close overlay modal with methods to capture action when modal is submitted.
+@blurb Shows a modal that overlays the screen
 @homepage index.html
 @demo index.html
 -->
 <dom-module id="px-modal">
   <template>
     <style include="px-modal-styles"></style>
 
-    <div on-tap="modalButtonClicked">
-      <content select=".modal-trigger"></content>
-    </div>
-    <div class="modal flex flex--top flex--center full-height invisible" id="{{modalId}}" role="dialog" aria-labelledby$="{{modalIdTitle(modalId)}}">
-      <section class="modal__content" role=region>
-        <template is="dom-if" if="{{modalHeading}}">
-          <h3 id="{{modalIdTitle(modalId)}}" class="modal__title epsilon weight--normal">{{modalHeading}}</h3>
-        </template>
-        <slot></slot>
-        <div class="modal__buttons flex flex--right">
-          <div class=flex>
-            <template class="modal-toggle" is="dom-if" if="{{btnModalNegative}}" restamp>
-              <button type="button" on-click="modalActionButtonClicked" id="btnModalNegative" class="btn">{{btnModalNegative}}</button>
-            </template>
-            <template class="modal-toggle" is="dom-if" if="{{btnModalPositive}}" restamp>
-              <button type="button" on-click="modalActionButtonClicked" id="btnModalPositive" class$="btn {{_getClass(btnModalPositiveDisabled)}}">{{btnModalPositive}}</button>
-            </template>
+    <div class="modal" id="modal" on-transitionend="_handleTransitionEnd">
+      <div class="modal__box">
+        <!-- Header -->
+        <div class="modal__header">
+          <slot name="header">
+            <h3 class="modal__header__text epsilon weight--normal">[[headerText]]</h3>
+          </slot>
+        </div>
+        <!-- Body -->
+        <div class="modal__body">
+          <slot name="body">
+            <p class="modal__body__text">[[bodyText]]</p>
+          </slot>
         </div>
-      </section>
+        <!-- Actions -->
+        <div class="modal__triggers">
+          <!-- Actions: Reject -->
+          <div class="modal__triggers__reject" on-click="_handleRejectTriggerTapped">
+            <slot name="reject-trigger">
+              <button type="button" class="btn" id="reject-trigger-button">[[rejectText]]</button>
+            </slot>
+          </div>
+          <!-- Actions: Accept -->
+          <div class="modal__triggers__accept" on-click="_handleAcceptTriggerTapped">
```

Comment:
Old code? There isn't a `modal__triggers__accept` in the sass file.

Reply:
Fixed